bigquery_etl/metadata/parse_metadata.py

```diff
@@ -157,6 +157,7 @@ class MonitoringMetadata:
     """Metadata for specifying observability and monitoring configuration."""
 
     enabled: bool = attr.ib(True)
+    collection: Optional[str] = attr.ib(None)
```

Comment:
In the Bigeye UI a metric can be added to multiple collections, and it's unfortunate that the `bigconfig` setup doesn't seem to allow for that.